tests/hf_hydrodata/test_gridded.py

```diff
@@ -1791,5 +1791,28 @@ def test_smap_current_conditions():
     assert (data[62] - 0.3409) <= 0.001, "Data for long block length not read properly"
 
 
+def test_cw3e_version():
+    """Test request for CW3E dataset using dataset_version parameter."""
```

Comment:
Oh, this is a really good point. Yes, things are set up such that if a user does not specify the version number then the latest version available will be used. I don't know how the filtering is affected by whether the user has access to all dataset versions or not, but I agree it would be safest to set things up such that the latest public version has the highest version number. I will update this now and also add a test case for when a user is not specifying `dataset_version` to test this prioritization explicitly.

Reply:
If you do not specify the dataset version is should default to the most recent version (largest version sorted by string name). I believe this was unit tested, but it would be good to test this with a real CW3E entries now that you have two version.

Reply:
@welytle - good comment. I will add this as a discussion point for our next full team software meeting. I think this default to 1.0 is what we want long-term but indeed could unexpectedly change user workflows in the short-term.